modules/metering-install-verify.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * metering/metering-installing-metering.adc
+
+[id="metering-install-verify_{context}"] 
+= Verify Installation 
+
+To verify your metering installation you can check that all the required pods have been activated, and check that your report data sources are beginning to import data.
+
+.Procedure
+
+. Navigate to *Workloads* -> *Pods* in the metering namespace and verify that pods are being created. This can take up to a minute after installing the metering stack.
+
```

Comment:
Everything past here isn't being indented properly with step 1. And then the following steps keep starting over the numbering at 1. You need to put "+" on empty lines to keep everything within a step together.